src/compiler/utilities.ts

```diff
@@ -7494,7 +7494,11 @@ namespace ts {
         }
     }
 
-        // Reserved characters, forces escaping of any non-word (or digit), non-whitespace character.
+    export function startsWithDirectory(path: string, dirPath: string): boolean {
+        return tryRemoveDirectoryPrefix(path, dirPath) !== undefined;

```

Comment:
Are the paths already in a form that accounts for platform case-sensitivity?

Reply:
Working on this in #24975

Reply:
Also, this PR never uses the function any more.